src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ClearNuGetLocalResourcesWindow.xaml.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.PlatformUI;
+using NuGet.PackageManagement.UI.ViewModels;
+using NuGet.VisualStudio;
+using NuGet.VisualStudio.Telemetry;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// Interaction logic for ClearNuGetLocalResourcesWindow.xaml
+    /// </summary>
+    public partial class ClearNuGetLocalResourcesWindow : DialogWindow
+    {
+        public ClearNuGetLocalResourcesWindow(ClearNuGetLocalsViewModel viewModel)
+        {
+            DataContext = viewModel;
+            InitializeComponent();
+        }
+
+        private void DialogWindow_ContentRendered(object sender, System.EventArgs e)
+        {
+            NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async delegate
```

Comment:
Why do you use ThreadHelper in some places and NuGetUIThreadHelper in others?

Reply:
No reason! Great catch. I noticed this at some point and forgot about it later 🤦‍♂️